onnxscript/optimizer/_constant_folding.py

```diff
@@ -154,6 +154,10 @@ def __init__(self):
         self._sym_value_map: dict[ir.Value, Any] = {}
         self._initializer_inputs: list[set[ir.Value]] = []
 
+    @property
+    def symbolic_value_map(self) -> dict[ir.Value, Any]:
```

Comment:
Is it possible to make the value type of the dictionary more precise?

Reply:
Done